src/scenes/Contacts/components/Map/Map.js

```diff
@@ -0,0 +1,49 @@
+import React, { useContext } from 'react';
+import { GlobalContext } from '@contexts';
+
+import styles from './Map.module.scss';
+
+const Map = () => {
+  const { imagesAPI } = useContext(GlobalContext);
+  const images = imagesAPI.get([
+    'contacts/contact-map.svg',
+    'contacts/flag-icon.svg',
+  ]);
+
+  const items = [
```

Comment:
This items should be transferred from component, it gets into the closure, and will not be created every time a component is updated